src/ceph-disk/ceph_disk/main.py

```diff
@@ -4677,8 +4700,8 @@ def unset_suppress(path):
     disk = os.path.realpath(path)
     if not os.path.exists(disk):
         raise Error('does not exist', path)
-    if not stat.S_ISBLK(os.lstat(path).st_mode):
-        raise Error('not a block device', path)
+    if not ldev_is_diskdevice(path):
```

Comment:
I think we can change path to disk here, for consistency. I don't see any useful use case where it would matter to use path instead of disk. This is mostly likely what was intended originally.